snapshots/er-lts-13.13.0.yaml

```diff
@@ -0,0 +1,152 @@
+resolver: lts-13.13
+name: er-lts.13.13.0.yaml
+
+packages:
+# Upstream 'master' addresses the issues we had in the PR
+# but it's not on stackage yet.
+# https://github.com/frontrowed/stratosphere/issues/111
```

Comment:
Can't you refer to the version on hackage?

```
extra-deps:
- stratosphere-0.34.0
```

Reply:
Does matter if this master commit SHA is exactly the same as that tag? Do you still prefer hackage version?

Reply:
Not especially, though it is possible to get prebuilt packages using hackage version? You probably know what's usual, better than me.

Reply:
AFAIK there are no pre-built **binary** packages with stack. It downloads source and compiles/caches locally. 
If I update this reference all that would practically change is the mechanics of how it gets source code (GH download vs tgz download from hackage).

Anyway, I guess there's some aesthetic value to this, so, I'll change it :)